tests/foreman/longrun/test_oscap.py

```diff
@@ -86,15 +97,30 @@ def content_view(module_target_sat, module_org):
 
 
 @pytest.fixture(scope='module', autouse=True)
-def activation_key(module_target_sat, module_org, lifecycle_env, content_view):
+def activation_key(module_target_sat, module_org, lifecycle_env):
     """Create activation keys"""
     repo_values = [
-        {'repo': settings.repos.satclient_repo.rhel8, 'akname': ak_name['rhel8']},
-        {'repo': settings.repos.satclient_repo.rhel7, 'akname': ak_name['rhel7']},
-        {'repo': settings.repos.satclient_repo.rhel6, 'akname': ak_name['rhel6']},
+        {
+            'repo': settings.repos.satclient_repo.rhel9,
+            'akname': ak_name['rhel9'],
+            'cvname': cv_name['rhel9'],
+        },
+        {
+            'repo': settings.repos.satclient_repo.rhel8,
+            'akname': ak_name['rhel8'],
+            'cvname': cv_name['rhel8'],
+        },
+        {
+            'repo': settings.repos.satclient_repo.rhel7,
+            'akname': ak_name['rhel7'],
+            'cvname': cv_name['rhel7'],
+        },
```

Comment:
Is this being too smart?
```suggestion
            'repo': getattr(settings.repos.satclient_repo, rhel),
            'akname': ak_name[rhel],
            'cvname': cv_name[rhel],
        } for rhel in ('rhel9', 'rhel8', 'rhel7')
```